### DISCUSSION TITLE
为什么2.0.0版本在win编译源码后找不到对应的 .tar.gz 文件呢

### DISCUSSION THREAD

**Discussion by HD0do** [apache/streampark#2411] [Q&A]
1.2.0版本编译后在 streampark-console\streampark-console-service\target是可以找到

**Comment by yaofengchn** [comment]
![image](https://user-images.githubusercontent.com/31480221/234162364-c4bad97c-f0b3-47ea-af4e-ea6c94c9a5c7.png)
我在mac下编译后是可以找到的，你应该是编译没成功

**Comment by caicancai** [comment]
I tried win compilation, it is ok, in the dist directory
